source/clear-linux/guides/network/firewalls.rst

```diff
@@ -0,0 +1,198 @@
+.. _firewalls:
+
+Firewalls
+#########
+
+Firewalls control access to and from systems based on network packet
+attributes like IP address, port, payload and more. 
+
+The `Netfilter <https://netfilter.org/>`_ framework in the Linux kernel
+performs packet filtering and provides the means for implementing a software
+firewall in Linux. |CL-ATTR| has a couple different firewall front-end options
+for managing the Linux firewall.
+
+
+.. contents:: :local:
+    :depth: 2
+   
+
+Default ruleset
+***************
+
+|CL| does not impose a firewall policy out of the box. All traffic is allowed
+inbound and all traffic is allowed outbound. However, `tallow`_ is installed
+by default and may dynamically create rules temporarily restricting SSH
+access.
+
+.. warning::
+
+   Changing firewall configuration can cause abrupt network disconnection, if
+   misconfigured. If this happens on a remote host, local recovery may be
+   required.
+   
+   Be sure to test your firewall configuration before committing it permanently.
```

Comment:
perhaps "and ensure your system will remain accessible remotely in case this is a requirement." ?